server/app/app.go

```diff
@@ -117,27 +116,44 @@ func (a *App) configure() error {
 }
 
 func (a *App) configureOTel() error {
-
-	traceExporter, err := otlptracegrpc.New(context.Background())
+	traceExporter, err := otlptracegrpc.New(
+		context.Background(),
+		otlptracegrpc.WithEndpoint(fmt.Sprintf("%s:%d", a.config.GetString("otlp.jaegerHost"), a.config.GetInt("otlp.jaegerPort"))),
+		otlptracegrpc.WithInsecure())
 
 	if err != nil {
 		a.log.Error("Unable to create a OTL exporter", err)
 		return err
 	}
 
-	traceResources := resource.NewWithAttributes(
-		semconv.SchemaURL,
-		semconv.ServiceNameKey.String(OTLServiceName),
-	)
-
 	traceProvider := tracesdk.NewTracerProvider(
+		// Always be sure to batch in production.
 		tracesdk.WithBatcher(traceExporter),
-		tracesdk.WithResource(traceResources),
+		// Record information about this application in a Resource.
+		tracesdk.WithResource(
+			resource.NewWithAttributes(
+				semconv.SchemaURL,
+				semconv.ServiceName(a.config.GetString("otlp.serviceName")),
```

Comment:
Do we need defaults for these otlp configs?

Reply:
I think it should be mandatory for prod/stag deployments. For local environment the local.yaml already have the defaults

Reply:
Got it!